src/services/MapBounds/MapBounds.js

```diff
@@ -110,3 +111,26 @@ export const boundsWithinDegrees = function(bounds1, bounds2, maxAllowedDegrees)
                Math.abs(normalizedBounds1.getNorth() - normalizedBounds2.getNorth()),
                Math.abs(normalizedBounds1.getSouth() - normalizedBounds2.getSouth())])
 }
+
+
+/**
+ * Calculates the best bounding box to incorporate all the given coordinates.
+ */
+export const calculateBoundingBox = function(coordinates) {
+  const lats = []
+  const lngs = []
+
+  _each(coordinates, c => {
+    lats.push(c[1])
+    lngs.push(c[0])
+  })
+
+  // calc the min and max lng and lat
+  const minlat = Math.min.apply(null, lats)
+  const maxlat = Math.max.apply(null, lats)
+  const minlng = Math.min.apply(null, lngs)
+  const maxlng = Math.max.apply(null, lngs)
+
+  // create a bounding rectangle that can be used in leaflet
+  return [minlat,minlng,maxlat,maxlng]
```

Comment:
Is this order correct? Typically it's WSEN. For example, here's what the leaflet docs say about its bboxString() method:

> Returns a string with bounding box coordinates in a 'southwest_lng,southwest_lat,northeast_lng,northeast_lat' format.